tsconfig.json

```
@@ -24,6 +24,10 @@
     "types": ["react", "node", "jest"],
     "newLine": "LF",
     "pretty": true,
+    "typeRoots": ["./@types"],
+    "paths": {
+      "@images/*": ["assets/images/*"]
+    }
   },
-  "esclude": ["node_modules"]
+  "esclude": ["node_modules"],
```

I believe there is a typo here:

```suggestion
  "exclude": ["node_modules"],
```